app/src/main/res/layout/fragment_edit_profile.xml

```diff
@@ -79,12 +78,11 @@
 
         </LinearLayout>
 
-        <androidx.appcompat.widget.AppCompatButton
-            android:id="@+id/buttonUpdate"
+        <com.google.android.material.button.MaterialButton
```

Comment:
Just use Button

Should work

Reply:
Are you sure? MaterialButton has different ripples and looks better IMO

Reply:
Is there a difference, the library should automatically inflate Button to MaterialButton

Reply:
Changed MaterialButton to Button in latest push